memento-cassandra.root/usr/share/clearwater/cassandra-schemas/memento.sh

```diff
@@ -1,36 +1,42 @@
 #! /bin/bash
-. /usr/share/clearwater/cassandra_schema_utils.sh
 
 cassandra_hostname="127.0.0.1"
 
 . /etc/clearwater/config
+. /usr/share/clearwater/utils/check-root-permissions 1
```

Comment:
We didn't check the permissions in any of the other files, why do we have to check in this one?

Reply:
We do in some - we should in all. I'll fix up the others to include this.